src/tests/Microsoft.PowerFx.Interpreter.Tests/Helpers/SemanticTokenTestHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation.
```

Comment:
There's already a helper for semantic tokens test. Just move this there

Reply:
Ah missed that one. Lets use that.